art/estimators/classification/pytorch.py

```diff
@@ -28,6 +28,7 @@
 from typing import Any, Dict, List, Optional, Tuple, Union, TYPE_CHECKING
 
 import numpy as np
+import torch
```

Comment:
Please move the `torch` import inside the method using it. This is necessary to allow using ART without PyTorch.